Core/Core/Helpers/Http.cs

```diff
@@ -128,23 +131,24 @@ public static async Task<bool> Ping(string hostnameOrAddress)
   }
 
   /// <summary>
-  /// Pings and tries gettign data from a specific address to verify it's online. Retries 3 times.
+  ///
```

Comment:
We lost the summary?

Reply:
I'll add a summary (tho, I didn't remove this one by accidently)

The comment about retries is (as far as I can tell) misleading/inaccurate.
And this is an internal function, so the comment is just for us.

But the importance is that this is a `HTTP Get` request (not a ICMP/ping). Worthy of a comment